backend/src/user/tests/user.e2e.spec.ts

```diff
@@ -104,6 +106,51 @@ describe("User", () => {
     });
   });
 
+  describe("POST /users/password", () => {
+    it(`update password`, async () => {
+      await request(app.getHttpServer())
+        .post("/users/password")
+        .send({
+          id: 1,
```

Comment:
The ID shouldn't be necessary and might throw an error if we enable input validation on tests.